### ISSUE TITLE
Start from BigWigs / bams

### ISSUE THREAD

**Issue by ferenckata** [ComputationalRegulatoryGenomicsICL/cageflow#39]
### Description of feature

Option to start the pipeline from mapped reads. Potentially, have an option to run only the mapping? We would like to have this feature, and might be useful for others: in case you explore mapping strategies before continuing but don't want to keep large amount of temp files

**Comment by ferenckata** [comment]
I think it would be fairly easy with creating a new channel reading in bigwigs, and start from nextflow run cager_main.nf
although I never tried

**Comment by sidorov-si** [comment]
I agree. It would be useful, apart from running the whole pipeline, to be able to start it from bigWigs (or BAM files after bowtie2 mapping) or to finish it just before the first CAGEr step. And it would also be convenient to be able to run the mapping part only. I also never tried but I think it should be relatively straightforward to create the corresponding options that will guide the choice of the first subworkflow to execute: `if (params.maponly) {...}`, etc.

**Comment by ferenckata** [comment]
I think this is closed now.
